charts/capsule/Chart.yaml

```diff
@@ -21,7 +21,7 @@ sources:
 
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
-version: 0.1.4
+version: 0.1.5
```

Comment:
Let's skip the version bump, this will be done when bumping to Capsule v0.1.1 release.

Reply:
done